public/ratings.js

```diff
@@ -1,22 +1,26 @@
-const ratingsDbUrl = 'http://localhost:3000/ratings';
 const $printArticleForRating = $('.printArticleForRating');
-var articles = {};
+const $RateArticleButton = $('#rate');
+let theArticle = null;
```

Comment:
what's this variable used for, out of curiosity?

Reply:
I thought I would need to use it outside of my function but I didn't. I've removed it and will do another commit.